README.md

```diff
@@ -9,6 +9,9 @@ We track bugs in Launchpad at
 https://bugs.launchpad.net/subiquity. Snappy first boot issues can
 also be discussed in the forum at https://forum.snapcraft.io.
 
+Our localization platform is Launchpad, translations are managed at
+https://translations.launchpad.net/subiquity
```

Comment:
except this is not correct, they are actually managed at https://translations.launchpad.net/ubuntu/+source/subiquity/

Reply:
I will create another one, sorry for that.